[{"critere": "1", "command": ["-h", "d\u00e9poser -h", "acheter -h", "vendre -h", "solde -h", "titres -h", "valeur -h", "projection -h"], "nom": "<p><h3>Commande Help", "erreurAttendu": [], "attendu": ["usage:"], "description": "<p><strong>V\u00e9rifier l'existence de la commande <code>help</code>.</strong></p>", "commentaire": "<p>La commande <code>help</code> n'existe pas dans un ou plusieurs cas.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": ["<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', '-h']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'd\u00e9poser', '-h']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'acheter', '-h']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'vendre', '-h']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'solde', '-h']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'titres', '-h']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'valeur', '-h']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'projection', '-h']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>"], "note": 60, "ponderation": 60}, {"critere": "1", "command": ["d\u00e9poser", "acheter", "vendre", "solde", "titres", "valeur", "projection", "d\u00e9poser --portefeuille", "d\u00e9poser --quantit\u00e9", "d\u00e9poser --date", "d\u00e9poser -q", "d\u00e9poser -d", "d\u00e9poser -p", "acheter --portefeuille", "acheter --quantit\u00e9", "acheter --titres", "acheter --date", "acheter -q", "acheter -t", "acheter -d", "acheter -p", "vendre --portefeuille", "vendre --quantit\u00e9", "vendre --titres", "vendre --date", "vendre -q", "vendre -t", "vendre -d", "vendre -p", "solde --graphique", "solde --portefeuille", "solde --date", "solde -d", "solde -p", "solde -g", "titres --graphique", "titres --portefeuille", "titres --titres", "titres --date", "titres -t", "titres -d", "titres -p", "titres -g", "valeur --graphique", "valeur --portefeuille", "valeur --date", "valeur --titres", "valeur -t", "valeur -d", "valeur -p", "valeur -g", "projection --graphique", "projection --portefeuille", "projection --titres", "projection --date", "projection --rendement", "projection --volatilit\u00e9", "projection -t", "projection -d", "projection -r", "projection -v", "projection -p", "projection -g"], "nom": "<p><h3>Nomenclature", "erreurAttendu": ["expected one argument", "expected at least one argument"], "attendu": [], "description": "<p><strong>V\u00e9rifer la nomenclature des arguments et actions.</strong></p>", "commentaire": "<p>Des actions et/ou arguments sont mal nomm\u00e9s.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": [], "note": 10, "ponderation": 10}, {"critere": "1", "command": ["-v", "-d", "-q", "-t", "-g", "-r", "-p", "d\u00e9poser -v", "d\u00e9poser -t", "d\u00e9poser -g", "d\u00e9poser -r", "acheter -v", "acheter -g", "acheter -r", "vendre -v", "vendre -g", "vendre -r", "solde -v", "solde -q", "solde -t", "solde -r", "titres -v", "titres -q", "titres -r", "valeur -v", "valeur -q", "valeur -r", "projection -q"], "nom": "<p><h3>Trop d'arguments", "erreurAttendu": ["unrecognized arguments", "following arguments are required"], "attendu": [], "description": "<p>V\u00e9rifier que les actions n'ont pas d'arguments superflus.</strong></p>", "commentaire": "<p>Des actions ont des arguments superflus.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": [], "note": 10, "ponderation": 10}, {"critere": "1", "command": ["d\u00e9poser", "acheter", "vendre", "solde", "titres", "valeur", "projection", "d\u00e9poser -q", "d\u00e9poser -d", "d\u00e9poser -p", "acheter -q", "acheter -t", "acheter -d", "acheter -p", "vendre -q", "vendre -t", "vendre -d", "vendre -p", "solde -d", "solde -p", "solde -g", "titres -t", "titres -d", "titres -p", "titres -g", "valeur -t", "valeur -d", "valeur -p", "valeur -g", "projection -t", "projection -d", "projection -r", "projection -v", "projection -p", "projection -g"], "nom": "<p><h3>Manque d'argument", "erreurAttendu": ["expected one argument", "expected at least one argument"], "attendu": [], "description": "<p><strong>V\u00e9rifier que les actions n'ont pas d'arguments manquants.</strong></p>", "commentaire": "<p>Des actions ont des arguments manquants.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": [], "note": 10, "ponderation": 10}, {"critere": "1", "command": ["d\u00e9poser -h", "projection -h"], "nom": "<h3>Metavar", "erreurAttendu": [], "attendu": [], "description": "<p><strong>V\u00e9rifier que les arguments ont le bon metavar.</p>", "commentaire": "<p>Des arguments n'ont pas le bon <code>metavar</code>.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": [], "note": 10, "ponderation": 10}, {"critere": "2", "command": ["d\u00e9poser -q 1000000000 -d 2018-09-17 -p PF1", "d\u00e9poser -q 10000 -d 2018-09-17", "d\u00e9poser -d 2018-09-17 -p PF0", "d\u00e9poser -q 10000 -p PF2"], "nom": "<p><h3>D\u00e9poser", "erreurAttendu": [], "attendu": [], "description": "<p><strong>V\u00e9rifier l'action <code>d\u00e9poser</code>.</strong></p>", "commentaire": "<p>L'action <code>d\u00e9poser</code> n'a pas le comportement attendu.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": [], "note": 16, "ponderation": 16}, {"critere": "2", "command": ["acheter -q 10000 -t goog -d 2018-09-17 -p PF1", "acheter -q 5 -t goog -d 2018-09-17", "acheter -t goog -d 2018-09-17 -p PF0", "acheter -q 5 -t goog -p PF2"], "nom": "<p><h3>Acheter", "erreurAttendu": ["Liquidit\u00e9Insuffisante"], "attendu": [], "description": "<p><strong>V\u00e9rifier l'action <code>acheter</code>.</strong></p>", "commentaire": "<p>L'action <code>acheter</code> n'a pas le comportement attendu.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": [], "note": 16, "ponderation": 16}, {"critere": "2", "command": ["vendre -q 500 -t goog -d 2018-09-17 -p PF1", "vendre -q 2 -t goog -d 2018-09-17", "vendre -q 500 -d 2018-09-17 -p PF1", "vendre -t goog -d 2018-09-17 -p PF0", "vendre -q 2 -t goog -p PF2"], "nom": "<p><h3>Vendre", "erreurAttendu": ["ErreurQuantit\u00e9"], "attendu": [], "description": "<p><strong>V\u00e9rifier l'action <code>vendre</code>.</strong></p>", "commentaire": "<p>L'action <code>vendre</code> n'a pas le comportement attendu.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": [], "note": 20, "ponderation": 20}, {"critere": "2", "command": ["solde -d 2018-09-17 -p PF1", "solde -d 2018-09-17", "solde -p PF0", "solde -p PF2"], "nom": "<p><h3>Solde", "erreurAttendu": [], "attendu": ["989595550.00", "6531.85", "1.00", "6971.77"], "description": "<p><strong>V\u00e9rifier l'action <code>solde</code>.</strong></p>", "commentaire": "<p>L'action <code>solde</code> n'a pas le comportement attendu.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": ["<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'solde', '-d', '2018-09-17', '-p', 'PF1']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'solde', '-d', '2018-09-17']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'solde', '-p', 'PF0']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'solde', '-p', 'PF2']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>"], "note": 16, "ponderation": 16}, {"critere": "2", "command": ["titres -t goog -d 2018-09-17 -p PF1", "titres -t goog -d 2018-09-17", "titres -d 2018-09-17 -p PF0", "titres -t goog -p PF2"], "nom": "<p><h3>Titres", "erreurAttendu": [], "attendu": ["[G|g][O|o][O|o][G|g]=9000", "[G|g][O|o][O|o][G|g]=3", "[G|g][O|o][O|o][G|g]=3", "[G|g][O|o][O|o][G|g]=3"], "description": "<p><strong>V\u00e9rifier l'action <code>titres</code>.</strong></p>", "commentaire": "<p>L'action <code>titres</code> n'a pas le comportement attendu.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": ["<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'titres', '-t', 'goog', '-d', '2018-09-17', '-p', 'PF1']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'titres', '-t', 'goog', '-d', '2018-09-17']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'titres', '-d', '2018-09-17', '-p', 'PF0']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'titres', '-t', 'goog', '-p', 'PF2']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>"], "note": 16, "ponderation": 16}, {"critere": "2", "command": ["valeur -t goog -d 2018-09-17 -p PF1", "valeur -t goog -d 2018-09-17", "valeur -t goog -p PF0", "valeur -d 2018-09-17 -p PF2"], "nom": "<p><h3>Valeur", "erreurAttendu": [], "attendu": ["10404450.00", "3468.15", "0.00", "0.00"], "description": "<p><strong>V\u00e9rifier l'action <code>valeur</code>.</strong></p>", "commentaire": "<p>L'action <code>valeur</code> n'a pas le comportement attendu.</p>", "sortie": ["RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : ", "RESULT : Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n", "ERREUR : "], "erreur": [], "mauvaisResultat": ["<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'valeur', '-t', 'goog', '-d', '2018-09-17', '-p', 'PF1']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'valeur', '-t', 'goog', '-d', '2018-09-17']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'valeur', '-t', 'goog', '-p', 'PF0']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>", "<li><p>Dans le contexte suivant :</p><pre class=\"line-numbers  language-python\"><code>['python3.7', '../unbundled/bundle-033-05.hg/gesport.py', 'valeur', '-d', '2018-09-17', '-p', 'PF2']</code></pre><p>Votre code ne soul\u00e8ve pas d'erreur mais ceci n'\u00e9tait pas le r\u00e9sultat attendu :<p><pre class=\"line-numbers  language-python\"><code>Traceback (most recent call last):\n  File \"../unbundled/bundle-033-05.hg/gesport.py\", line 11, in <module>\n    from portefeuille import Portefeuille\nModuleNotFoundError: No module named 'portefeuille'\n</code></pre></li>"], "note": 16, "ponderation": 16}]